modules/transaction-payment/src/lib.rs

```diff
@@ -751,13 +751,21 @@ where
 		}
 	}
 
-	fn weight_to_fee(weight: Weight) -> PalletBalanceOf<T> {
+	pub fn weight_to_fee(weight: Weight) -> PalletBalanceOf<T> {
 		// cap the weight to the maximum defined in runtime, otherwise it will be the
 		// `Bounded` maximum of its data type, which is not desired.
 		let capped_weight = weight.min(T::BlockWeights::get().max_block);
 		T::WeightToFee::calc(&capped_weight)
 	}
 
+	/// Apply multiplier to fee, return the final fee. If multiplier is `None`, use
+	/// `next_fee_multiplier`.
+	#[allow(dead_code)]
+	fn apply_multiplier_to_fee(fee: PalletBalanceOf<T>, multiplier: Option<Multiplier>) -> PalletBalanceOf<T> {
```

Comment:
this and weight_to_fee should be exposed via a trait otherwise other pallets won't be able to use it